src/net/sourceforge/kolmafia/persistence/ItemFinder.java

```diff
@@ -277,7 +325,9 @@ private static void filterNameList(List<String> nameList, Match filterType) {
           itemId != -1
               && !ItemDatabase.getAttribute(
                   itemId, EnumSet.of(Attribute.TRADEABLE, Attribute.MATCHABLE, Attribute.QUEST))
-              && !NPCStoreDatabase.contains(itemId));
+              && !NPCStoreDatabase.contains(itemId)
+              && !InventoryManager.itemAvailable(itemId)
+              && ConcoctionDatabase.getIngredients(itemId).length == 0);
```

Comment:
near as I can tell, this checks whether an item can be created, not whether it can be created from items available to the user. This may be what is desired. Am I correct, and was it?

Reply:
Yeah, I want these checks to be conservative about culling the list. If we find there to be a problem or annoyance later we can adjust.